### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by joshhopkins** [taigaio/taiga-front#1160]
More a question/potential feature request:

What are Taiga users doing to create a product roadmap? 

- I've used projects in the past to divide up product stages i.e MVP, beta, public beta etc. But this can get convoluted very quickly. 
- Sprints are too short sighted and time critical.
- Epics can be used, but feel like a work around.
- Tags?

Having a road map view which bundles together specific Epics to a timeline would be great to display a global vision to whole team. Thoughts?

**Comment by Xaviju** [comment]
The idea behind EPICS is to bundle multiple User Stories that represent a bigger functionality, release, idea.

> An EPIC story is a large undefined user story that must be subdivided.

https://blog.taiga.io/epic-stories.html

Looks exactly what you are looking for. Why does it feels like a workaround?

**Comment by joshhopkins** [comment]
> **But aren’t EPICS a way to group large functional blocks of the project, For example an API, the front part, Mobile?**
> 
> No. But you can call these "modules" or something similar to that, but these modules don't have any specific value

From my understanding:

- A product timeline doesn't have specific value
- Should be tied to rough ETA's (timeline)
- Contain/group Epics

In my mind the hierarchy looks like:

1. MVP = Milestone on roadmap
2. Sign up flow = Epic
3. As a user I need to be able to verify my account via mobile = US
4. Implement SMS verification using Twilio = Task

Perhaps the simplest solution is to group/filter Epics using tags?

**Comment by joshhopkins** [comment]
Any more thoughts on this?

This article makes some interesting points on roadmaps + agile – https://www.atlassian.com/agile/roadmaps

Jira Portfolio seems to handle this quite nicely – https://www.atlassian.com/software/jira/portfolio

**Comment by LeGiangAnh** [comment]
Waiting for roadmap feature as well

**Comment by hdave** [comment]
Jira roadmaps seems like a good idea on paper, the reality however is that the tool is almost worthless.  That said, I would like Taiga to make an attempt at this functionality -- by doing it the Taiga Way.  Start very simple and enhance it overtime based on carefully thought out consideration.

For now, a simple list of releases that act as a grouping mechanism for Epics and Stories would be satisfactory.  Each release could have a name, description, tags and list of milestones with estimated dates.  IMO, this covers 80% of what you need.  

In lieu of this functionality, we tags epics and stories with a release tag (1.1, 1.2, etc.) and searching on the release tag gives you a decent summary of what is included in the release.

**Comment by joshhopkins** [comment]
> For now, a simple list of releases that act as a grouping mechanism for Epics and Stories would be satisfactory. Each release could have a name, description, tags and list of milestones with estimated dates. IMO, this covers 80% of what you need.

@hdave agree. The roadmap should be a simple way of giving Epics date/time values, so the whole team can easily see high-level project milestones.

In terms of UI, I'd really like to see a horizontal timeline (gantt like) where a user can create releases/milestones (which contain epics), or simply drag and drop epics directly from a list – this would keep things nice a flexible. These objects could reflect basic Epics metadata, such as Progress etc.

**Comment by joshhopkins** [comment]
Could admin please re-label this as 'enhancement'?

**Comment by bameda** [comment]
Relabeled, but we use https://tree.taiga.io/project/taiga/ to manage the Taiga project and not the github issues system. This issues is tagged as enhancement a lot of time ago (https://tree.taiga.io/project/taiga/issue/4765)

**Comment by Mike-EEE** [comment]
In my case I am looking for a simple way of chaining a bunch of major features together given a ballpark estimate of hours for each one and getting a nice overlay of what that looks like from a big picture sort of deal.  Hygger is very close to this but they do not allow public URLs and also their estimated hours do not impact the End date.  Go figure.

So, an Excel spreadsheet it is, then. 😝

**Comment by luisfergromo** [comment]
Hey TEAM, im also waiting for Roadmaps. Right now im using Taiga but we need roadmaps to view at High Level the project. I know there's a lot of work but please develop this feature, could be awesome to used early.

**Comment by AlexAtkinson** [comment]
Hello. I'd also like to see a roadmap feature. See monday.com, roadmonk, aha.io, etc., for reference.
Atlassian acquired Agilecraft purely to get their roadmaping expertise. Roadmapping is essential to framing up timelines, and can't be underestimated.

I would love it if you could add 'flags' on the roadmap that represent millstone completions, with mouseover, or drilldown functionality to show the completion status of the epics that make up the milestones. Additional analysis of drilldown rate vs deadline for issuing a health-status of the milestones would be truly awesome.

Thanks

**Comment by LauridsJMikkelsen** [comment]
Any update on this one? maybe in relation to TaigaNext? 
FYI This github issue is the first links that comes up when I google "Taiga roadmap".
